website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -0,0 +1,392 @@
+  only supported Type is "OIDC." 
+
+- `TokenLocality` `(string: "")` - Defines whether the ACL Auth Method
+  creates a local or global token when performing SSO login. This field must be
+  set to either "local" or "global"
+
+- `MaxTokenTTL` `(duration: <required>)` - Defines the maximum life of a token created
+  by this method. When set it will initialize the `ExpirationTime` field on all
+  tokens to a value of `Token.CreateTime + AuthMethod.MaxTokenTTL`. This field is
+  not persisted beyond its initial use. Can be specified in the form of `"60s"` or
+  `"5m"` (i.e., 60 seconds or 5 minutes, respectively). 
+
+- `Default` `(bool: false)` - Defines whether this ACL Auth Method is to be
+  set as default when running `nomad login` command. 
+
+- `Config` `(ACLAuthMethodConfig: nil)` - The raw configuration to use for
+  the auth method.
+
+  - `OIDCDiscoveryURL` `(string: "")` - The OIDC Discovery URL, without
+    any .well-known component (base path).
+  
+  - `OIDCClientID` `(string: "")` - The OAuth Client ID configured with
+    your OIDC provider.
+
+  - `OIDCClientSecret` `(string: "")` - The OAuth Client Secret
+    configured with your OIDC provider.
+
+  - `BoundAudiences` `(array<string>)` - List of aud claims that are valid for
+    login; any match is sufficient.
+
+  - `AllowedRedirectURIs` `(array<string>)` - A list of allowed values for
+    redirect_uri. Must be non-empty.
+  
+  - `DiscoveryCaPem` `(array<string>)` - PEM encoded CA certs for use by the TLS
+    client used to talk with the OIDC Discovery URL. If not set, system
+    certificates are used.
+
+  - `SigningAlgs` `(array<string>)` - A list of supported signing algorithms.
+    Defaults to `RS256`.
+
+  - `ClaimMappings` `(map[string]string)` - Mappings of claims (key) that will
+    be copied to a metadata field (value). Use this if the claim you are capturing
+    is singular (such as an attribute).
+
+    When mapped, the values in each list can be any of a number, string, or
+    boolean and will all be stringified when returned.
+
+  - `ListClaimMappings` `(map[string]string)` - Mappings of claims (key) will be
+    copied to a metadata field (value). Use this if the claim you are capturing is
+    list-like (such as groups).
+
+### Sample Payload
+
+```json
+{
+    "name": "example-acl-auth-method",
+    "type": "OIDC",
+    "tokenlocality": "global",
+    "maxtokenttl": "1h0m0s",
+    "default": true
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request POST \
+    --header "X-Nomad-Token: <NOMAD_TOKEN_SECRET_ID>" \
+    --data @payload.json \
+    https://localhost:4646/v1/acl/auth-method/example-acl-auth-method
+```
+
+### Sample Response
+
+```json
+{
+    "MaxTokenTTL": "1h0m0s",
+    "Name": "example-acl-auth-method",
+    "Type": "OIDC",
+    "TokenLocality": "global",
+    "Default": true,
+    "Config": null,
+    "CreateTime": "0001-01-01T00:00:00Z",
+    "ModifyTime": "0001-01-01T00:00:00Z",
```

Comment:
I think it would be nice to have these as non-default time.Time values, to make it feel a little more real.